measurements/regard/regard.py

```diff
@@ -0,0 +1,178 @@
+        `max_data_regard` and 'max_references_regard': the maximum regard for each group  (if `aggregation` = `maximum`)
+
+Examples:
+
+Example 1 (single input):
+>>> regard = evaluate.load("regard")
+>>> group1 = ['xyz are described as mean', 'xyz are thought of as being too ambitious']
+>>> results = regard.compute(data = group1)
+>>> for d in results['regard']:
+...     print({l['label']: round(l['score'],2) for l in d})
+{'neutral': 0.95, 'positive': 0.02, 'negative': 0.02, 'other': 0.01}
+{'negative': 0.97, 'other': 0.02, 'neutral': 0.01, 'positive': 0.0}
+
+Example 2 (comparison mode):
+>>> regard = evaluate.load("regard", "compare")
+>>> group1 = ['xyz are described as mean', 'xyz are thought of as being too ambitious']
+>>> group2 = ['xyz are known for making too much noise', 'xyz are described as often violent']
+>>> results = regard.compute(data = group1, references = group2)
+>>> print({k: round(v, 2) for k, v in results['regard_difference'].items()})
+{'neutral': 0.46, 'positive': 0.01, 'negative': -0.46, 'other': -0.01}
+
+Example 3 (returns the maximum regard score per category):
+>>> regard = evaluate.load("regard", "compare")
+>>> group1 = ['xyz are described as mean', 'xyz are thought of as being too ambitious']
+>>> group2 = ['xyz are known for making too much noise', 'xyz are described as often violent']
+>>> results = regard.compute(data = group1, references = group2, aggregation = "maximum")
+>>> print({k: round(v, 2) for k, v in results['max_data_regard'].items()})
+{'neutral': 0.95, 'positive': 0.02, 'negative': 0.97, 'other': 0.02}
+>>> print({k: round(v, 2) for k, v in results['max_references_regard'].items()})
+{'negative': 0.98, 'other': 0.04, 'neutral': 0.03, 'positive': 0.0}
+
+Example 4 (returns the average regard score):
+>>> regard = evaluate.load("regard", "compare")
+>>> group1 = ['xyz are described as mean', 'xyz are thought of as being too ambitious']
+>>> group2 = ['xyz are known for making too much noise', 'xyz are described as often violent']
+>>> results = regard.compute(data = group1, references = group2, aggregation = "average")
+>>> print({k: round(v, 2) for k, v in results['average_data_regard'].items()})
+{'neutral': 0.48, 'positive': 0.01, 'negative': 0.5, 'other': 0.01}
+>>> print({k: round(v, 2) for k, v in results['average_references_regard'].items()})
+{'negative': 0.96, 'other': 0.02, 'neutral': 0.02, 'positive': 0.0}
+"""
+
+
+def regard(group, regard_classifier):
+    group_scores = defaultdict(list)
+    group_regard = regard_classifier(group)
+    for pred in group_regard:
+        for pred_score in pred:
+            group_scores[pred_score["label"]].append(pred_score["score"])
+    return group_regard, dict(group_scores)
+
+
+@evaluate.utils.file_utils.add_start_docstrings(_DESCRIPTION, _KWARGS_DESCRIPTION)
+class Regard(evaluate.Measurement):
+    def _info(self):
```

Comment:
I would do a check if the config name is either compare or default. checkout the `glue` metrics for an example of how to do this.

Reply:
otherwise if you mistype compare you will silently get the default.

Reply:
like this? (line 121)